health-services/plan-service/src/main/java/digit/web/models/PlanFacilityRequest.java

```diff
@@ -0,0 +1,30 @@
+package digit.web.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.Valid;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.egov.common.contract.request.RequestInfo;
+import org.springframework.validation.annotation.Validated;
+
+/**
+ * PlanFacilityRequest
+ */
+@Validated
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class PlanFacilityRequest {
+    @JsonProperty("RequestInfo")
+    @Valid
+    private RequestInfo requestInfo = null;
+
+    @JsonProperty("PlanFacility")
+    @Valid
+    private PlanFacility planFacility = null;
+
+
+}
```

Comment:
the current design does not enforce @NotNull annotations for these properties because the application logic is structured to handle potential null values gracefully.